src/main/java/frc/robot/subsystems/drive/DriveIOCTRE.java

```diff
@@ -52,6 +55,8 @@ public void updateInputs(DriveIOInputs inputs) {
     inputs.failedDaqs = state.FailedDaqs;
 
     inputs.gyroRate = super.getPigeon2().getAngularVelocityZWorld().getValue();
+    inputs.gyroConnected = gyroConnectedDebounce.calculate(super.getPigeon2().isConnected());
```

Comment:
why do we have to debounce here? and is this really worth doing once we are driving? if you lose the gyro .. i mean how is that relevant to the driver? if anything shouldn't the system automatically switch the driving type or something else? what does one do in the middle of a match when the gyro alert goes off?

Reply:
I hope this helps answer your questions.

1. Why do we have to debounce here? - To eliminate false positives, as we got some without them. Maybe we can add the debounce on the other side of the IO and make it so there is a warning if triggered no matter what but then an error for debouncer? Thoughts??? Open to suggestions on this.
2. Do you think this is worth doing once we are driving? - I believe so. I don't see why you would not change it. The biggest thing is to log the value to check if something goes wrong during the match. Setting the Alert isn't super important, but it would be helpful to see before a match starts or if doing a system check on your robot.
3. Shouldn't the system automatically switch the driving type or something else? I believe the CTRE code switches to using encoder values to compensate if the gyro goes out.
4. What does one do during a match when the gyro alert goes off? Recognize that any aligned features will not work. This is much more useful during 99% of your season. When testing the robot and something stops working, you can look at the Alert tab and see what isn't working (same thing for encoders, drive and steer motors).

Reply:
weird that the gyro gives false positivies on a "connected" method.. i'd think that goes into the can level and the bus is aware of it being there or not.. 
maybe that indicates poor wiring so that'd be a bonus

and that'd be the reason i'd want to see the alert while driving.. .. poor connections! or loose jiggly bobgs.

Reply:
Okay, that sounds good. The issue mostly occurred on boot-up. I'll merge this but then open a new issue to switch all Alerts to send a warning if before debouncer then error on debouncer.